.cirrus.yml

```diff
@@ -162,80 +121,89 @@ fmt_task:
         - make fmt
         - ./hack/tree_status.sh
 
-# Test building of new cache-images for future PR testing, in this PR.
-# Output images will be stored only for a very short time, then automatically deleted.
-test_cache_images_task:
 
-    only_if: >-
-        $CIRRUS_BRANCH != 'master' &&
-        $CIRRUS_CHANGE_MESSAGE =~ '.*\*\*\*\s*CIRRUS:\s*TEST\s*IMAGES\s*\*\*\*.*'
+# Build the static binary
+static_binary_task:
+    depends_on:
+        - 'config'
+        - 'fmt'
 
-    # VMs created by packer are not cleaned up by cirrus, must allow task to complete
-    auto_cancellation: $CI != "true"
+    script:
+        - dnf install -y podman make git gcc
+        - make containerized
 
-    gce_instance:
-        image_name: "${IMAGE_BUILDER_CACHE_IMAGE_NAME}"
-        cpu: 4
-        memory: "4Gb"
-        disk: 200
-        # Additional permissions for building GCE images, within a GCE VM
-        scopes:
-            - 'compute'
-            - 'devstorage.full_control'
+    binaries_artifacts:
+        path: "bin/conmon"
```

Comment:
Do we need this conmon stuff?

Reply:
Ooops!  Never mind.  Too many reviews, I thought I was looking at "common", not "conmon"....  <FacePalm>

Reply:
not strictly speaking, no.  It simply archives a copy of the binary, in case it's needed for some hind-sight debugging.  I can remove it it's getting in the way.

Reply:
it's gone.